src/components/ModalEdit/index.js

```diff
@@ -0,0 +1,133 @@
+  const { name: selectedDoctorName, id: selectedDoctorId } = doctor;
+  date = new Date(date);
+  const [day, month, year] = [
+    date.getDate() < 10 ? `0${date.getDate()}` : date.getDate(),
+    date.getMonth() + 1 < 10 ? `0${date.getMonth() + 1}` : date.getMonth() + 1,
+    date.getFullYear(),
+  ];
+
+  const cancel = () => {
+    setItemToEdit({});
+    setErrors(false);
+  };
+
+  return (
+    <>
+      <Modal show={true} backdrop="static" keyboard={false}>
+        <Modal.Header>
+          <Modal.Title>Edit visit</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <div>
+            <div>
+              <label htmlFor="patientName">Patient name</label>
+            </div>
+            <input
+              id="patientName"
+              type="text"
+              name="patient_name"
+              placeholder="Patient name"
+              className="w-100 p-2 my-2"
+              value={patient_name}
+              onInput={({ target }) =>
+                setEditedVisitPartials({ patient_name: target.value })
+              }
+            />
+          </div>
+          <div>
+            <div>
+              <label htmlFor="doctorName">Doctor name</label>
+            </div>
+            <select
+              className="w-100 p-2 my-2"
+              id="doctorName"
+              onChange={({ target }) =>
+                setEditedVisitPartials({ doctor_id: parseInt(target.value) })
+              }
+            >
+              <option value={selectedDoctorId}>
+                ---{selectedDoctorName}----
+              </option>
+              {doctors.length &&
+                doctors.map((doctor, index) => {
+                  const { name, id, specialty } = doctor;
+                  return (
+                    <option key={`doctor-${index}`} value={id}>
+                      {name} ({specialty})
+                    </option>
+                  );
+                })}
+            </select>
+          </div>
+          <div>
+            <div>
+              <label htmlFor="date">Date</label>
+            </div>
+            <DatePicker
+              className="w-100 p-2 my-2"
+              name="date"
+              placeholderText="dd.mm.yyyy"
+              id="date"
+              dateFormat="dd.MM.yyyy"
+              value={`${day}.${month}.${year}`}
+              onChange={(date) => setEditedVisitPartials({ date })}
+            />
+          </div>
+          <div>
+            <div>
+              <label htmlFor="complaints">Complaints</label>
+            </div>
+            <textarea
+              type="text"
+              name="text"
+              placeholder="Write your complaints here"
+              id="complaints"
+              className="w-100 p-2 my-2"
+              value={text}
+              onInput={({ target }) =>
+                setEditedVisitPartials({ text: target.value })
+              }
+            />
+          </div>
+          {errors &&
```

Comment:
it's not the first time I've seen similar error code duplicated. maybe it's better to put it in a component?